pkg/virt-controller/watch/snapshot/restore.go

```diff
@@ -98,6 +98,10 @@ func (ctrl *VMRestoreController) updateVMRestore(vmRestoreIn *snapshotv1.Virtual
 
 	logger.V(1).Infof("Updating VirtualMachineRestore")
 
+	if !VmRestoreProgressing(vmRestoreIn) {
```

Comment:
consider renaming and changing the logic of the function to VMRestoreComplete since is checks for the 'Complete' variable and not for the, and so you will also not need to check the negative of Progressing

Reply:
Yeah, I can understand that, but I think I in general I prefer the phrase "not progressing" as there may be multiple reasons to not reconcile, not just completeness.  Even though that is all we care about here